CONTRIBUTING.md

```diff
@@ -70,7 +77,7 @@ Reviewers will run through roughly the following checklist:
 - Are boundary cases considered?
 - Are error cases handled?
 
-(Once we have some tests, the reviewer will check that all the tests are passing,
+(the reviewer will check that all the tests are passing,
```

Comment:
> I guess travis checks to see that the tests are run.

For sure, but *humans* need to verify that the tests make sense, are reasonably independent of one another (so that one bug doesn't cause lots of seemingly-unrelated tests to fail), and that the most important or tricky/important corners have a little extra thorough coverage.